packs/core-abilities/interstellar-shift.json

```diff
@@ -20,11 +20,30 @@
           "target": "self",
           "unit": "m"
         },
-        "description": "<p>Effect: The user freely Moves up to 4m of a Movement Type of its choice and chance Forme. Depending on its Forme, it gains another Ability.</p><p>- Normal Forme - Pressure</p><p>- Attack Forme - Mold Breaker</p><p>- Speed Forme - Speed Boost</p><p>- Defense Forme - Sturdy",
-        "img": "icons/svg/explosion.svg"
+        "description": "<p>Effect: The user freely Moves up to 6m of a Movement Type of its choice and change Forme, healing @Tick[2] of HP. Depending on its Forme, it gains another Ability.</p><p>- Normal Forme - Pressure</p><p>- Attack Forme - Mold Breaker</p><p>- Speed Forme - Speed Boost</p><p>- Defense Forme - Sturdy</p>",
```

Comment:
```suggestion
        "description": "<p>Effect: The user freely Moves up to 6m of a Movement Type of its choice and change Forme, healing @Tick[2] of HP. Depending on its Forme, it gains another Ability.</p><ul><li>Normal Forme: @UUID[Compendium.ptr2e.core-abilities.Item.RmLYmSTJEvyBNTh5]{Pressure}</li><li>Attack Forme: @UUID[Compendium.ptr2e.core-abilities.Item.i2YkuytUf6z6fm34]{Mold Breaker}</li><li>Speed Forme: @UUID[Compendium.ptr2e.core-abilities.Item.LpovBWFJb0J2ggv1]{Speed Boost}</li><li>Defense Forme: @UUID[Compendium.ptr2e.core-abilities.Item.uOMBIIsKTRROmTBW]{Sturdy}</li></ul>",
```

Reply:
@titaniumdragoonneo not sure if you were editing this on git/foundry, but you can make bullet point lists native in Foundry/HTML using `<ul><li>`. The Foundry Rich Text Editor supports this fine.

I've also replaced all ability names with links to said abilities, nice lil QoL.